test/e2e/common/fixtures.py

```diff
@@ -0,0 +1,112 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# 	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+"""Common SageMaker test fixtures.
+"""
+
+import pytest
+
+from e2e import (
+    CONFIG_RESOURCE_PLURAL,
+    MODEL_RESOURCE_PLURAL,
+    ENDPOINT_RESOURCE_PLURAL,
+    DATA_QUALITY_JOB_DEFINITION_RESOURCE_PLURAL,
+    create_sagemaker_resource,
+    wait_sagemaker_endpoint_status,
+)
+
+from e2e.replacement_values import REPLACEMENT_VALUES
+from acktest.resources import random_suffix_name
+from acktest.k8s import resource as k8s
+
+
+@pytest.fixture(scope="module")
+def xgboost_churn_endpoint(sagemaker_client):
```

Comment:
Could you separate out the model and endpoint creation  such that we can use model other places such as the transform job ?

Reply:
we already have a helper method create_sagemaker_resource which is generic, everything else is specific to a use case because the replacements and spec file names are hardcoded.

If you find a transform job example in future that uses this xgboost_churn model, we can move it out easily

will keep as-is for now

Reply:
I am using almost similar code to create the transform job model object. If we parameterized the s3 path, code could be re-used right away.

Reply:
resolving this, I will make this change as a part of my PR if it actually dedups some logic.